cherrypy/__init__.py

```diff
@@ -57,7 +57,7 @@
 """
 
 try:
-    import pkg_resources
+    import importlib
 except ImportError:
     pass
```

Comment:
If you're going to use `importlib.metadata` below, you'll want to `import importlib.metadata`.

Reply:
Bah, what a dumb mistake. I'll upload a fixed patch.